SixTrack/sixtrack.s

```diff
@@ -59293,13 +59293,13 @@ c$$$            endif
      &'|        |           |     [M]   |     [MM]   |          |',     &
      &'          |        |             |     |ORD.|',                  &
      &'    [%]  |      [%]  |'/t1,125('-'))
-10010 format(t1,'|',i8,'|X ',f9.5,'|X ',f9.4,'|X ',f10.6,'|',d10.4, '|',&
-     &d10.4,'|',f8.4,'|X ',d11.5,'|X ',i3,'| ',i2,' |X ', f7.3,'|X+Y ', &
+10010 format(t1,'|',i8,'|X ',f9.5,'|X ',f9.4,'|X ',f10.6,'|',d11.4, '|',&
```

Comment:
I'm, pretty sure that format 10010 should allign with format 10000 just above it?